untitled/main/com/aca/homework/week13/race/condition/list/List.java

```diff
@@ -0,0 +1,12 @@
+package com.aca.homework.week13.race.condition.list;
+
+public interface List<E> {
+
+    void add(E item);
+
+    int size();
+
+    E get(int index);
```

Comment:
this method was not in requirement

Reply:
I had to add it so I could write a @Test

Reply:
I had to add it to be able to write a @Test